builtin/credential/github/path_login.go

```diff
@@ -177,6 +171,26 @@ func (b *backend) verifyCredentials(ctx context.Context, req *logical.Request, t
 		client.BaseURL = parsedURL
 	}
 
+	if config.OrganizationID == 0 {
+		// Previously we did not verify using the Org ID. So if the Org ID is
+		// not set, we will trust-on-first-use and set it now.
+		err = config.setOrganizationID(ctx, client)
+		if err != nil {
+			b.Logger().Error("failed to set the organization_id on login", "error", err)
+			return nil, nil, err
+		} else {
+			entry, err := logical.StorageEntryJSON("config", config)
+			if err != nil {
+				return nil, nil, err
+			}
+
+			if err := req.Storage.Put(ctx, entry); err != nil {
```

Comment:
I _think_ that this should be fine, but this means that if a login is performed on a performance standby (for enterprise) that trips this storage write case the login request will be routed to the active node.